src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -23,6 +23,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @ToString(exclude = {"users"})
+@EqualsAndHashCode(exclude = {"users", "locationList", "skills"})
```

Comment:
It will be good way if you will exclude this fields form @ToString as well.

Reply:
done